### PR TITLE
svt-av1: 1.2.1 -> 1.4.1

### PR DESCRIPTION
**Pull request by Madouura**
###### Description of changes
Version bump.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Madouura** [comment]
Result of `nixpkgs-review pr 207624` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>kdenlive</li>
    <li>obs-studio-plugins.obs-hyperion</li>
  </ul>
</details>
<details>
  <summary>5 packages failed to build:</summary>
  <ul>
    <li>obs-studio-plugins.obs-backgroundremoval</li>
    <li>obs-studio-plugins.obs-ndi</li>
    <li>olive-editor</li>
    <li>python310Packages.dm-sonnet</li>
    <li>python39Packages.dm-sonnet</li>
  </ul>
</details>
<details>
  <summary>50 packages built:</summary>
  <ul>
    <li>MIDIVisualizer</li>
    <li>arcanPackages.all-wrapped</li>
    <li>arcanPackages.arcan</li>
    <li>arcanPackages.arcan-wrapped</li>
    <li>arcanPackages.cat9-wrapped</li>
    <li>arcanPackages.durden-wrapped</li>
    <li>arcanPackages.ffmpeg</li>
    <li>arcanPackages.pipeworld-wrapped</li>
    <li>arcanPackages.prio-wrapped</li>
    <li>arcanPackages.xarcan</li>
    <li>corrscope</li>
    <li>ffmpeg_5-full</li>
    <li>handbrake</li>
    <li>haruna</li>
    <li>imagination</li>
    <li>jellyfin</li>
    <li>jellyfin-ffmpeg</li>
    <li>libsForQt5.kdenlive</li>
    <li>libsForQt5.soundkonverter</li>
    <li>liquidsoap</li>
    <li>manim</li>
    <li>media-downloader</li>
    <li>megasync</li>
    <li>obs-studio</li>
    <li>obs-studio-plugins.input-overlay</li>
    <li>obs-studio-plugins.looking-glass-obs</li>
    <li>obs-studio-plugins.obs-gstreamer</li>
    <li>obs-studio-plugins.obs-move-transition</li>
    <li>obs-studio-plugins.obs-multi-rtmp</li>
    <li>obs-studio-plugins.obs-nvfbc</li>
    <li>obs-studio-plugins.obs-pipewire-audio-capture</li>
    <li>obs-studio-plugins.obs-source-record</li>
    <li>obs-studio-plugins.obs-vkcapture</li>
    <li>obs-studio-plugins.wlrobs</li>
    <li>printrun</li>
    <li>python310Packages.moderngl-window</li>
    <li>python310Packages.pydub</li>
    <li>python310Packages.pyglet</li>
    <li>python310Packages.pytmx</li>
    <li>python310Packages.tensorflow-datasets</li>
    <li>python39Packages.moderngl-window</li>
    <li>python39Packages.pydub</li>
    <li>python39Packages.pyglet</li>
    <li>python39Packages.pytmx</li>
    <li>python39Packages.tensorflow-datasets</li>
    <li>restream</li>
    <li>sunshine</li>
    <li>svt-av1</li>
    <li>video-trimmer</li>
    <li>webcamoid</li>
  </ul>
</details>

**Comment by Madouura** [comment]
None of the errors from ``nixpkgs-review`` seem to be related to the change.
``Install_NDI_SDK_v5_Linux`` isn't downloading something correctly.
``onnxruntime`` failed because of nddl. (i.e.: ``'struct dnnl::memory::desc' has no member named 'dims'``)
``olive-editor`` failed because of something to do with qt: (i.e: ``aggregate 'QPainterPath path' has incomplete type and cannot be defined``)
``dm-sonnet`` tries to import check ``tensorflow``.

**Comment by risicle** [comment]
`nixpkgs-review` reveals no new failures, macos 10.15

**Comment by ck3d** [comment]
the onnxruntime compilation error was reported in https://github.com/NixOS/nixpkgs/issues/207551

**Comment by Madouura** [comment]
I should mention I tested this with ffmpeg on a few videos to make sure it __actually__ works and it indeed does.

**Review by ck3d — APPROVED** [review]

**Comment by Madouura** [comment]
cc @SuperSandro2000 for merge

File: pkgs/tools/video/svt-av1/default.nix [hunk 1]
```diff
@@ -1,20 +1,35 @@
...
+  ];
+
+  passthru.updateScript = gitUpdater {
+    rev-prefix = "v";
+  };
 
   meta = with lib; {
+    homepage = "https://gitlab.com/AOMediaCodec/SVT-AV1";
     description = "AV1-compliant encoder/decoder library core";
+
```
Comment (SuperSandro2000) [inline pkgs/tools/video/svt-av1/default.nix:32]: There should not be blank lines here

Comment (Madouura) [reply pkgs/tools/video/svt-av1/default.nix:32]: Is there a standard for this in nixpkgs?
I personally think a (newline) separation between single and multi-line declarations is more readable.

Comment (ajs124) [reply pkgs/tools/video/svt-av1/default.nix:32]: > Is there a standard for this in nixpkgs?

there isn't. there isn't even a recommendation. whatever anyone comments is their personal preference.

there's https://github.com/NixOS/rfcs/pull/101, but that's still not merged.

**Pull request merged**